src/shared/HtmlReady.js

```diff
@@ -146,8 +146,8 @@ function link(state, child) {
 
             // Unlink potential phishing attempts
             if (
-                child.textContent.match(/(www\.)?steemit\.com/) &&
-                !url.match(/https?:\/\/(.*@)?(www\.)?steemit\.com/)
+                child.textContent.match(/(www\.)?steemit\.com/i) &&
```

Comment:
Outside the ambit of this work but I would suggest a more aggressive phish detector to cover someting like:
**Steemit is so cool**: [www.steemit.cool](http://www.lotsofmany.com/)
**New british steemit**: [www.steemit.co.uk](http://www.eggalone.com/)
Suggest:
```
child.textContent.match(/(steemit)+\.([\w\d\.])*/i)
```

Reply:
too many false positives w/ that approach

Reply:
they are using `steemiit.com` IRL: https://steemit.com/pretty/@siennawu/snow-day#@soniia/re-siennawu-snow-day-20180218t140601956z